contrib/tablefunc/tablefunc.control

```diff
@@ -1,5 +1,5 @@
 # tablefunc extension
 comment = 'functions that manipulate whole tables, including crosstab'
-default_version = '1.0'
+default_version = '1.1'
```

Comment:
why exact this number?

Reply:
I didn't find information about extension versioning rules at postgres doc, so at this patch approach from the [postgres citext extension](https://github.com/postgres/postgres/tree/master/contrib/citext) was used.